terragrunt/org_account/organization/organizations.tf

```diff
@@ -30,29 +30,32 @@ resource "aws_organizations_organizational_unit" "AFT" {
   parent_id = local.root
 }
 
+resource "aws_organizations_policy_attachment" "AFT-cds_snc_universal_guardrails" {
+  policy_id = aws_organizations_policy.cds_snc_universal_guardrails.id
+  target_id = aws_organizations_organizational_unit.AFT.id
+}
+
 
 resource "aws_organizations_organizational_unit" "DumpsterFire" {
   name      = "DumpsterFire"
   parent_id = local.root
 }
 
+resource "aws_organizations_policy_attachment" "DumpsterFire-cds_snc_universal_guardrails" {
+  policy_id = aws_organizations_policy.cds_snc_universal_guardrails.id
+  target_id = aws_organizations_organizational_unit.DumpsterFire.id
+}
 
 resource "aws_organizations_organizational_unit" "Production" {
   name      = "Production"
   parent_id = local.root
 }
 
-
 resource "aws_organizations_organizational_unit" "Sandbox" {
   name      = "Sandbox"
   parent_id = local.root
 }
 
-resource "aws_organizations_organizational_unit" "SandboxMigration" {
```

Comment:
Just because I'm curious like a 🐱 what was this OU used for?

Reply:
It was to fix the CT upgrade kerfuffle... had to move all accounts out of the OU, fix the drift that somehow got introduced and broke enrolment of accounts... Managed to get all things sorted but had left the OU in case we'd need it again soon. 😄

Reply:
Thank you!  appreciate the explanation